src/DistributedMetadata/MetadataService.cpp

```diff
@@ -284,8 +297,15 @@ void MetadataService::startup()
         {
             LOG_INFO(log, "Detects the current log has `{}` role", this_role);
 
-            /// First try to create corresponding dwal
-            doCreateDWal(kctx);
+            try
+            {
+                /// First try to create corresponding dwal
+                doCreateDWal(kctx);
+            }
+            catch (Exception e)
+            {
+                LOG_INFO(log, e.message());
```

Comment:
when failed to create dwal, we log at info level and ignore the error ?

Reply:
Every time the service restarts, it will check whether the __system_ddls topic is created here